lib/cdo/create_header.rb

```diff
@@ -0,0 +1,57 @@
+# This class provides the contents for the create projects
+# dropdown in the header.
+
+class CreateHeader
+  def self.get_create_dropdown_contents(options)
+    everyone_entries = [
+      {
+        id: "create_dropdown_playlab",
+        title: I18n.t("nav.create_button.playlab"),
+        url: CDO.studio_url('projects/playlab/new'),
+        image: "/shared/images/fill-70x70/courses/logo_playlab.png"
```

Comment:
idea: you could factor out the common part of the image URL and put it in the code at the end of this function.

Reply:
Oooh. I like that idea. I'll incorporate in my next PR.